card/mod/pointer/set/abstract/01_pointer.rb

```diff
@@ -75,9 +75,14 @@ def default_item_view
   end
 
   view :core do
+    voo.items[:view] = params[:item] if params[:item]
```

Comment:
So, the way this is _supposed_ to work is via `main_nest_items_opt`, which processes params[:item].  The intended benefit over a solution like this one is that the item view will only be applied to the main nest and not to any random rendering.  (Eg, if a CSS sheet had lots of nests in it).

But I think the problem is that this is not rendered as part of a nest at all.  That's a problem with our "main_nest" approach, I guess.  Maybe we need to broaden that to "main_content" and have it called in cases like this where the main content is not in a nest?